bgpd/bgp_ecommunity.h

```diff
@@ -326,7 +326,7 @@ static inline void encode_node_target(struct in_addr *node_id,
 	memset(eval, 0, sizeof(*eval));
 	eval->val[0] = ECOMMUNITY_ENCODE_IP;
 	if (!trans)
-		eval->val[0] |= ECOMMUNITY_ENCODE_IP_NON_TRANS;
+		eval->val[0] = ECOMMUNITY_ENCODE_IP_NON_TRANS;
```

Comment:
If this is not an `|`, then why overwrite the value we set at line 327? should this be 
```
	eval->val[0] = trans? ECOMMUNITY_ENCODE_IP : ECOMMUNITY_ENCODE_IP_NON_TRANS;
```

Reply:
This or that, IMO, no difference :)

Reply:
Changed.